### PR TITLE
add doc about cni performance test

### PR DESCRIPTION
**Pull request by Jeanine-tw**
@qiupingdai

### CODE REVIEW

File: docs/zh/navigation.yml [hunk 1]
```diff
@@ -436,6 +436,8 @@ nav:
           - 规划:
               - 网卡规划: network/plans/ethplan.md
               - 网络规划: network/plans/netplan.md
+          - 性能报告
+              - 网络 CNI 性能测试报告: network/performance/CNI-performance.md
```
Comment (Michelle951) [inline docs/zh/navigation.yml:440]: pls use small letters, according to our file naming conventions https://github.com/DaoCloud/DaoCloud-docs#naming-conventions

Comment (Jeanine-tw) [reply docs/zh/navigation.yml]: > pls use small letters, according to our file naming conventions https://github.com/DaoCloud/DaoCloud-docs#naming-conventions

fixed

Comment (Michelle951) [reply docs/zh/navigation.yml]: You need also to change the file name. Tips: only changing upper cases to small ones does not work here; you should also make additional minor changes to make it easier for computer programms to check the difference.

Comment (Jeanine-tw) [reply docs/zh/navigation.yml]: > You need also to change the file name. Tips: only changing upper cases to small ones does not work here; you should also make additional minor changes to make it easier for computer programms to check the difference.

thanks for your reminder!

**Review by windsonsea — APPROVED** [review]

**Pull request merged**